src/Microsoft.Azure.ServiceBus/ServiceBusDiagnosticsSource.cs

```diff
@@ -0,0 +1,771 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    using Microsoft.Azure.ServiceBus.Diagnostics;
+
+    internal class ServiceBusDiagnosticSource
+    {
+        public const string DiagnosticListenerName = "Microsoft.Azure.ServiceBus";
+
+        public const string ExceptionEventName = "Microsoft.Azure.ServiceBus.Exception";
+        public const string ProcessActivityName = "Microsoft.Azure.ServiceBus.Process";
+        public const string ProcessActivityStartName = "Microsoft.Azure.ServiceBus.Process.Start";
```

Comment:
`ProcessActivityStartName`, `ProcessSessionActivityName`, and `ProcessSessionActivityStartName` don't seem to be used.